Power Up Proto/src/org/usfirst/frc/team5493/robot/Robot.java

```diff
@@ -53,7 +62,7 @@ public void robotInit() {
 	 * the robot is disabled.
      */
     public void disabledInit(){
-
+    	//encoder.reset();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)